src/components/MoveModal.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { useI18n } from 'cozy-ui/react/I18n'
+import useInstanceSettings from 'hooks/useInstanceSettings'
+import { withClient } from 'cozy-client'
+
+import { Button } from 'cozy-ui/transpiled/react/Button'
+import { IllustrationDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+
+const importImage = require('assets/images/moved-cozy.svg')
+
+export const MoveModal = ({ client }) => {
+  const { t } = useI18n()
+  const { data: instanceSettings } = useInstanceSettings(client)
```

Comment:
By talking with Patrick, I have discovered that `useInstanceSettings` will make a request to the server for each component where it is used (my intuition was the opposite, maybe because the name does include `fetch`). The `useInstanceSettings` should be modified to use a named query and a fetch policy, IIRC.